doc/developers/contributing.rst

```diff
@@ -498,50 +503,51 @@ underestimate how easy an issue is to solve!
 .. _contribute_documentation:
 
 Documentation
--------------
+=============
 
 We are glad to accept any sort of documentation: function docstrings,
 reStructuredText documents (like this one), tutorials, etc. reStructuredText
 documents live in the source code repository under the ``doc/`` directory.
 
 You can edit the documentation using any text editor, and then generate the
-HTML output by typing ``make html`` from the ``doc/`` directory. Alternatively,
-``make`` can be used to quickly generate the documentation without the example
-gallery. The resulting HTML files will be placed in ``_build/html/stable`` and are viewable
-in a web browser. See the ``README`` file in the ``doc/`` directory for more information.
+HTML output by typing ``make`` from the ``doc/`` directory. Alternatively,
+``make html`` may be used to generate the documentation **with** the example
+gallery (which takes quite some time). The resulting HTML files will be
+placed in ``_build/html/stable`` and are viewable in a web browser.
 
 
 Building the documentation
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------
+
+First, make sure you have :ref:`properly installed <install_bleeding_edge>`
+the development version.
 
 Building the documentation requires installing some additional packages::
 
-    pip install sphinx sphinx-gallery numpydoc matplotlib Pillow pandas scikit-image joblib
+    pip install sphinx sphinx-gallery numpydoc matplotlib Pillow pandas scikit-image
 
 To build the documentation, you need to be in the ``doc`` folder::
 
     cd doc
 
-It also requires having the version of scikit-learn installed that corresponds
-to the documentation, e.g.::
```

Comment:
Why remove this? The command should be executed before cd ing into the doc folder though.

Reply:
we start the section with "First, make sure you have :ref:`properly installed <install_bleeding_edge>`
the development version."